packages/gui/src/@types/WebSocketService.ts

```diff
@@ -0,0 +1,12 @@
+type WebSocketService = {
+  connect: () => string; // returns connection ID
+  send: (connectionId: string, data: string) => void;
```

Comment:
Should be `(connectionId: string, data: string) => Promise<void>`

Reply:
updated